external_services/lti.py

```diff
@@ -75,27 +57,55 @@ def __init__(self, service, user, instance, host, title, context_id=None, link_i
             "context_label": course.code,
 
             "launch_presentation_locale": get_language(),
+            "launch_presentation_document_target": "window",
+            "launch_presentation_return_url": request.scheme + '://' + request.get_host() + instance.get_absolute_url(),
 
-            "tool_consumer_instance_guid": host + "/aplus",
+            "tool_consumer_instance_guid": request.get_host() + "/aplus",
             "tool_consumer_instance_name": "A+ LMS",
         })
 
         if service.api_access:
             self.parameters.update({
-                # FIXME: we need request or full host with protocol here!
-                'custom_context_api': '//' + host + api_reverse("course-detail", kwargs={'course_id': instance.id}),
+                'custom_context_api': request.scheme + '://' + request.get_host() + api_reverse("course-detail", kwargs={'course_id': instance.id}),
```

Comment:
Above comment applies here too :)

Reply:
This is a non-standard, custom parameter used to provide the URL to the A+ REST API. The host  could be defined in settings (for easier container setup), but are we doing it now or later?

Reply:
Later. Just commented as on general information.